normalize.py

```diff
@@ -20,38 +20,60 @@ def getval(self, x, y):
         val = self.array[int(x)][int(y)]
         return val
 
-    def getarray (self):
+    def getarray(self):
         return self.array
 
-#Takes image as parameter and normalizes it
-#Normalize means taking an array of value and making min 0 and max 100 (or whatever value you want to the max)
-#Returns normalized array
+
+# Takes image as parameter and normalizes it
+# Normalize means taking an array of value and making min 0 and max 100 (or whatever value you want to the max)
+# Returns normalized array
```

Comment:
Please reference the issues your code addressing in the comments, for documentation's sake.

e.g.:  
Ref: Req 2.0
Ref: Req 2.1

Reply:
Alright, i will make a different pull request that will address this issue for the rest of my code.

Reply:
Understood. Pull request accepted.